deployment/helm/nri-plugins/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+appVersion: main
+description: |
+  NRI resource policy is a NRI plugin that will apply hardware-aware resource allocation
+  policies to the containers running in the system.
+name: nri-plugins
```

Comment:
´nri-resource-policy` instead? Just thinking that we prolly don't put all future plugins under this chart